content/index.pl.md

```diff
@@ -0,0 +1,252 @@
+{:/}
+
+## Spojrzenie na dostępność {#context}
+
+<blockquote class="pull">
+  <p>Siła Internetu tkwi w jego uniwersalności.<br />
+    Dostęp dla każdego bez względu na niepełnosprawność jest jego zasadniczym atrybutem.</p>
+  <footer><cite>Tim Berners-Lee, Dyrektor W3C i twórca World Wide Web</cite></footer>
+</blockquote>
+
+Internet został zasadniczo zaprojektowany tak, aby służył wszystkim, niezależnie od ich urządzeń, oprogramowania, języka, lokalizacji czy sprawności. Gdy Internet osiąga ten cel, jest dostępny dla osób z różnorodnym zakresem zdolności słuchowych, ruchowych, wzrokowych i poznawczych.
+
+Dlatego wpływ niepełnosprawności zmienia się radykalnie w Internecie, bowiem Internet usuwa bariery w komunikacji i uczestnictwie, z którymi wielu ludzi mierzy się w świecie fizycznym. Z drugiej strony, gdy strony internetowe, aplikacje, technologie lub narzędzia są źle zaprojektowane, mogą stawiać bariery uniemożliwiające ludziom korzystanie z Internetu.
+
+**Dostępność jest niezastąpiona dla programistów i organizacji, które chcą tworzyć wysokiej jakości strony i narzędzia internetowe, zamiast wykluczać ludzi z korzystania ze swoich produktów i usług.**
+
+## Czym jest dostępność internetowa {#what}
+
+Dostępność internetowa oznacza, że ​​strony internetowe, narzędzia i technologie są projektowane i rozwijane w taki sposób, by mogły z nich korzystać osoby z niepełnosprawnościami. Mówiąc dokładniej, ludzie mogą:
+
+-   postrzegać, rozumieć, nawigować i wchodzić w interakcję z Internetem
+-   współtworzyć Internet. 
+
+Dostępność internetowa obejmuje wszystkie rodzaje niepełnosprawności wpływające na dostęp do Internetu, w tym niepełnosprawność:
+
+-   słuchu,
+-   poznawczą,
+-   neurologiczną,
+-   fizyczną,
+-   mowy,
+-   wzroku.
+
+Dostępność Internetu przynosi korzyści także osobom _bez_ niepełnosprawności, na przykład:
+
+-   osobom korzystającym z telefonów komórkowych, inteligentnych zegarków, inteligentnych telewizorów i&nbsp;innych urządzeń z małymi ekranami, różnymi trybami wprowadzania danych itp.
+-   osobom starszym ze zmieniającymi się zdolnościami ze względu na starzenie się,
+-   osobom z „tymczasową niepełnosprawnością”, np. złamaną ręką lub zgubionymi okularami,
+-   osobom oganiczonym okolicznościami, np. jasnym światłem słonecznym lub w otoczeniu, w którym nie mogą słuchać dźwięku,
+-   osobom korzystającym z wolnego połączenia internetowego albo mającym ograniczony lub kosztowny dostęp do internetu.
+
+Zobacz 7-minutowy film z przykładami tego, jak niezbędna jest dostępność dla osób z niepełnosprawnościami i jak przydatna dla wszystkich w rozmaitych sytuacjach:<br>
+{% include video-link.html title="Oblicza dostępności internetowej <em>(YouTube)</em>" href="https://www.youtube.com/watch?v=3f31oufqFSM" src="/content-images/wai-intro-accessibility/video-still-accessibility-perspectives-16-9.jpg" %}
+
+{::nomarkdown}
+{% include box.html type="start" h="3" title="Więcej o dostępności" class="simple aside" %}
```

Comment:
Original content: "More info on What is Accessibility"

The translation reads "More about accessibility". Would you think of a closer translation?

Reply:
```suggestion
{% include box.html type="start" h="3" title="Więcej informacji o dostępności" class="simple aside" %}
```

Reply:
@zwiastunsw Sorry for not being specific enough. I was thinking making it clearer it is about "What is" accessibility (when other sections are about how important it is/how to make the Web accessible, etc.).

Reply:
Ok :) I send PR